gradle/versions.gradle

```diff
@@ -1,29 +1,29 @@
-/*
- * Base project version and versions of common dependencies.
- */
-
-version = '2.14.0-SNAPSHOT'
-

```

Comment:
am not sure which kind of line endings we want to have here @szarnekow @kthoms any idea